src/Kernel32/Kernel32.Helpers.cs

```diff
@@ -256,6 +257,293 @@ public static ArraySegment<byte> ReadFile(SafeObjectHandle hFile, int nNumberOfB
             return new ArraySegment<byte>(buffer, 0, bytesRead);
         }
 
+        /// <summary>
+        ///     Marks any outstanding I/O operations for the specified file handle. The function only cancels I/O operations
+        ///     in the current process, regardless of which thread created the I/O operation.
+        /// </summary>
+        /// <param name="hFile">A handle to the file.</param>
+        /// <param name="lpOverlapped">
+        ///     A pointer to an <see cref="NativeOverlapped" /> data structure that contains the data used for asynchronous I/O.
+        ///     <para>If this parameter is NULL, all I/O requests for the hFile parameter are canceled.</para>
+        ///     <para>
+        ///         If this parameter is not NULL, only those specific I/O requests that were issued for the file with the
+        ///         specified
+        ///         <paramref name="lpOverlapped" /> overlapped structure are marked as canceled, meaning that you can cancel one
+        ///         or more requests, while the CancelIo function cancels all outstanding requests on a file handle.
+        ///     </para>
+        /// </param>
+        /// <returns>
+        ///     If the function succeeds, the return value is nonzero. The cancel operation for all pending I/O operations issued
+        ///     by the calling process for the specified file handle was successfully requested. The application must not free or
+        ///     reuse the <see cref="NativeOverlapped" /> structure associated with the canceled I/O operations until they have
+        ///     completed. The thread can use the GetOverlappedResult function to determine when the I/O operations themselves have
+        ///     been completed.
+        ///     <para>
+        ///         If the function fails, the return value is 0 (zero). To get extended error information, call the
+        ///         <see cref="GetLastError" /> function.
+        ///     </para>
+        ///     <para>
+        ///         If this function cannot find a request to cancel, the return value is 0 (zero), and
+        ///         <see cref="GetLastError" />
+        ///         returns <see cref="Win32ErrorCode.ERROR_NOT_FOUND" />.
+        ///     </para>
+        /// </returns>
```

Comment:
Do you know you don't have to copy this from the other overload? 
You can replace it all with this:
```cs
/// <inheritdoc cref="CancelIoEx(SafeObjectHandle, OVERLAPPED*)" />
```

Then, if you hover over your new method's name or its parameters, you'll see that the docs all magically show up.